onesignalclient/notification.py

```diff
@@ -59,34 +87,35 @@ def contents(self):
 
     @contents.setter
     def contents(self, value):
-        self._validate_content_dict(value)
-        self._contents = json.dumps(value)
+        if isinstance(value, str):
+            value = json.loads(value)
 
-    @property
-    def content_available(self):
-        return self._content_available
+        if not isinstance(value, dict):
+            raise TypeError('Value must be a dict.')
 
-    @content_available.setter
-    def content_available(self, value):
-        self._content_available = value
+        if not value.get(self.DEFAULT_LANGUAGE, False):
+            raise KeyError('Default language (%s) must be included.' % (
+                self.DEFAULT_LANGUAGE))
+
+        self._contents = json.dumps(value)
 
     @property
     def headings(self):
         return json.loads(self._headings)
 
     @headings.setter
     def headings(self, value):
-        self._validate_content_dict(value)
```

Comment:
Done and changed it so it returns the value

Reply:
Maybe we could change it's name, since it has a changed a bit. What do you think about renaming it to `_validate_and_return_content_dict`? Or maybe some other name that represents it usage in a better way?